### DISCUSSION TITLE
Is Fedora not supported anymore?

### DISCUSSION THREAD

**Discussion by spectkiller** [jc141x/jc141-bash#31] [General]
I can't find BindToInterface Package from jc141 rpmrepo and hence I can't play this game.

**Comment by PoorPocketsMcNewHold** [comment]
Answered here. https://github.com/orgs/jc141x/discussions/27#discussioncomment-7074065
They haven't been able to update their copr repo with DwarFS.
https://copr.fedorainfracloud.org/coprs/jc141/DwarFS/
You could try to compile it yourself, or use the prebuilt binaries of the projects, I had success with those on Solus. https://github.com/mhx/dwarfs#prebuilt-binaries

No idea how to help you on that BindToInterface package however.

**Reply by spectkiller** [reply]
thanks, the prebuilt binaries work just fine

**Comment by adminx01** [comment]
the bindtointerface package is not mandatory. it is optional

please use the matrix channel for support instead. you're not going to get much help here

**Reply by spectkiller** [reply]
yeah, I realized that later. Thanks for the help.

**Comment by hackernoodle** [comment]
we need a copr maintainer then we can officially support it again

**Comment by hackernoodle** [comment]
done; https://copr.fedorainfracloud.org/coprs/jc141/DwarFS/package/dwarfs/
